osu.Game/IO/Archives/ZipArchiveReader.cs

```diff
@@ -27,15 +31,12 @@ public override Stream GetStream(string name)
             if (entry == null)
                 throw new FileNotFoundException();
 
-            // allow seeking
-            MemoryStream copy = new MemoryStream();
+            var owner = MemoryAllocator.Default.Allocate<byte>((int)entry.Size);
```

Comment:
Wait... This isn't even using the I# allocator? O_o

Reply:
I think that’s an oversight with namespacing.m with the `HighPerformance` using…

Reply:
I've double checked and it is. Can you explain where you're seeing that it isn't?

Reply:
Ah indeed it is. `Toolkit.HighPerformance` has its own `MemoryOwner<>` that I was confusing with...
![image](https://user-images.githubusercontent.com/1329837/171077524-e03519ee-6deb-4fad-b6d4-dbd324ebf192.png)